docs/conf.py

```diff
@@ -25,7 +25,13 @@
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-# autodoc_mock_imports = ["digitalio", "busio"]
+autodoc_mock_imports = [
+    "busio",
+    "digitalio",
+    "displayio",
+    "microcontroller",
+    "terminalio",
+]
```

Comment:
Is there a reason you are using autodoc_mock_imports? I think all of these are already in requirements, so there's no need to mock the modules up.

Reply:
I've trimmed the list down in the most recent push, but if I don't include `displayio` and `terminalio` the docs won't build.

Reply:
That's odd considering both `displayio` and `terminalio` are part of this repo.

Reply:
Give me a while to investigate further. I've just realised that the generated docs are wrong.